.github/workflows/test.yml

```diff
@@ -104,6 +105,14 @@ jobs:
           { config: "Release", plat: "linux", os: "ubuntu-24.04", arch: "x64", tls: "quictls", systemcrypto: "-UseSystemOpenSSLCrypto", build: "-Test" },
           { config: "Debug", plat: "linux", os: "ubuntu-24.04", arch: "x64", tls: "quictls", systemcrypto: "-UseSystemOpenSSLCrypto", build: "-Test", xdp: "-UseXdp" },
           { config: "Release", plat: "linux", os: "ubuntu-24.04", arch: "x64", tls: "quictls", systemcrypto: "-UseSystemOpenSSLCrypto", build: "-Test", xdp: "-UseXdp" },
+          { config: "Debug", plat: "linux", os: "ubuntu-22.04", arch: "x64", tls: "openssl", sanitize: "-Sanitize", build: "-Test" },
+          { config: "Debug", plat: "linux", os: "ubuntu-22.04", arch: "x64", tls: "openssl", systemcrypto: "-UseSystemOpenSSLCrypto", sanitize: "-Sanitize", build: "-Test" },
```

Comment:
Can building with the latest openssl work with dynamic linking of the inbox openssl for libcrypto? And a higher-level question: Do we want to support the mix static/dynamic linking for openssl? I don't think so. Ideally, I'd rather support complete static link or complete dynamic link.

Reply:
hmm, I suppose the most correct answser to that is that it depends on the version of openssl provided by the distibution.  for ubuntu-22.04, the answer would be no, as its on 3.2.something.

I'll revert this